Runtime/Scripts/Internal/FFIClient.cs

```diff
@@ -122,13 +126,13 @@ private static void InitializeSdk()
             const bool captureLogs = false;
 #endif
 
-            NativeMethods.LiveKitInitialize(FFICallback, captureLogs);
+            NativeMethods.LiveKitInitialize(FFICallback, captureLogs, "unity", ""); // TODO: Get SDK version
```

Comment:
I verified this works for now but I'd love to get SDK version in. it's just not clear _where_ to get it from at runtime? Might need to inject it into a script like we do for the node SDK. any ideas @cloudwebrtc @theomonnom ?

Reply:
I found this but don't know if it works as expected
https://discussions.unity.com/t/how-get-packageinfo-by-name-and-version/762057/8

```C#
using UnityEngine;
using System.Linq;
using UnityEditor.PackageManager;

// ... within some class ...

private PackageInfo GetPackageInfo(string packageName)
{
    return UnityEditor.AssetDatabase.FindAssets("package")
        .Select(UnityEditor.AssetDatabase.GUIDToAssetPath)
            .Where(x => UnityEditor.AssetDatabase.LoadAssetAtPath<TextAsset>(x) != null)
        .Select(PackageInfo.FindForAssetPath)
            .Where(x => x != null)
        .First(x => x.name == packageName);
}
```

```
string packageName = "com.unity.cinemachine";
Debug.Log(GetPackageInfo(packageName).version);
```

Reply:
From what I saw online, `UnityEditor` isn't available at runtime on-device, only when running from the Unity Editor? Do you think that's accurate?

Reply:
gonna just punt on this

Reply:
Yeah, you are right, maybe we can add a version.cs, but we must check that the version number in version.cs is consistent with the version number in package.json before releasing a new version

Reply:
in the node sdk we dynamically generate a version.js file as part of the build process, but i'm not sure there's a clean way to pull that off here.

Reply:
if the Unity build process can generate version.cs, I think it is a good idea.